go/libkb/config.go

```diff
@@ -706,3 +706,7 @@ func (f *JSONConfigFile) SetTimeAtPath(path string, t keybase1.Time) error {
 func (f JSONConfigFile) GetLocalTrackMaxAge() (time.Duration, bool) {
 	return f.GetDurationAtPath("local_track_max_age")
 }
+
+func (f JSONConfigFile) GetMountDir() string {
+	return f.GetTopLevelString("mountdir")
```

Comment:
mount dir is 2 words I guess... so this should probably be "mount_dir" (snake case is used for config)